lib/tasks/rabbitmq.rake

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+QUEUES = %w(
+  git_clone
+  git_pull
+  hets\ >=\ 0.100.0
+  indexing
+  mailers
+  post_process_hets
+  process_commit
+)
+
+namespace :rabbitmq do
+  desc "Purge all rabbitmq queues"
+  task :purge do
```

Comment:
Not 100% sure, but I think that this needs the `:environment` because it uses `Settings`.

Reply:
Nope, I checked; none of them need `:environment`, not even for the database connection.

Reply:
Might have changed because of the whole `rails ...` instead of `rake ...`